styles/layout.tsx

```diff
@@ -17,4 +17,14 @@ export const layoutStyles = StyleSheet.create({
     flexDirection: 'row',
     flexWrap: 'wrap',
   },
+  settingContainer: {
+    marginTop: 30,
+    padding: 16,
+  },
+  sidebar: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    marginTop: 16,
```

Comment:
Don't we need to specify `display: 'flex'` too?

Reply:
display is by default flex here, so not needed.